src/vip/batch_address_test_tool/cloud_store.clj

```diff
@@ -0,0 +1,29 @@
+(ns vip.batch-address-test-tool.cloud-store
+  (:require [amazonica.aws.s3 :as s3]
+            [clj-time.core :as time]
+            [clj-time.coerce :as coerce]))
+
+(def conf {:endpoint "us-east-1"})
+
+(defn fetch-file
+  "Retrieves a given file from s3"
+  [bucket-name file-name]
+  (-> (s3/get-object conf :bucket-name bucket-name :key file-name :range 0)
+    :input-stream
+    slurp))
+
+
+(defn save-file
+  "Saves the upload file on s3 in the bucket with the give file name"
+  [bucket-name file-name upload-file]
+  (s3/put-object conf
+                 :bucket-name bucket-name
+                 :key file-name
+                 :file upload-file))
+
+(defn url-for-file
```

Comment:
It might be convenient to mention what you're returning; there's a `(.toString (url-for-file ,,,))` in v.b-a-t-t.processor later on.